src/foolscap/test/test_banana.py

```diff
@@ -1627,18 +1786,17 @@ def testConstrainedBool(self):
                       schema.BooleanConstraint(True))
 
 
-class A:
+# PY3KPORT: Let these test classes inherit from Bar as it
+# already contains our Py2 & 3 portable code for comparison/equality
+# Makes the code simpler.
+class A(Bar):
```

Comment:
...but it also means there no "old-style" class in the test-suite now, right?

Reply:
Nope. Class A inherits from Bar which is an old style class in Python2, so there are still old style classes in the test suite. 

```
>>> type(test_banana.A)
<type 'classobj'>
>>> type(test_banana.B)
<type 'classobj'>
```